.changeset/tricky-months-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@microsoft/atlas-css": minor
+---
+
+For browsers that support target-text, update the text color to make text and URLs more visible when highlighted
```

Comment:
Sorry one last little nit here, ` update the text color` should probably say ` update the background color`

Reply:
(you can just edit this file directly instead of using the changesets cli)